examples/cpy/Test_KitchenSink/metadata.json

```
@@ -0,0 +1,65 @@
+{
+    "name": "Kitchen sink component example",
+    "nickname": "Everything",
+    "category": "Componentizer",
+    "subcategory": "ALL",
+    "description": "This is an example with the everything and the kitchen sink in it, just to show all available options.",
+    "exposure": 4,
+    "instanceGuid": "cdd47086-f902-4b77-825b-6b79c3aaecc1",
+    "ghpython": {
+        "marshalGuids": true,
+        "iconDisplay": 2,
+        "inputParameters": [
+            {
+                "name": "X",
```

It seems something weird happens with these names: if this is capitalized, the variable will actually be renamed:
Using capital in the name here modified the actual variable to get
```suggestion
                "name": "x",
```